ref/general/LDAP-user-registries-authentication-authorization.adoc

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description: You can configure your applications that run on Open Liberty to manage authentication with a Lightweight Directory Access Protocol (LDAP) user registry.
+:page-layout: general-reference
+:seo-title: Configuring an LDAP user registry for authentication and authorization
+:seo-description: You can configure your applications that run on Open Liberty to manage authentication with a Lightweight Directory Access Protocol (LDAP) user registry.
+:page-layout: general-reference
+:page-type: general
+= LDAP user registries for authentication and authorization
+
+You can configure your applications that run on Open Liberty to manage authentication with a Lightweight Directory Access Protocol (LDAP) user registry.
+
+LDAP is an open protocol that supports authentication and authorization services.
+LDAP specifies the language that application servers use to communicate with user registries that store security information such as usernames, passwords, and user groups.
```

Comment:
"LDAP specifies the language that application servers use to communicate with user registries that store security information such as usernames, passwords, and user groups."
to
"LDAP specifies the language that application servers use to communicate with user registries, which store security information such as usernames, passwords, and user groups."